### ISSUE TITLE
User interface does not show up if using OpenGL Renderer

### ISSUE THREAD

**Issue by sebescudie** [benkuper/Chataigne#169]
**Environment**
 - OS: Windows 10 21H2
 - Version: 1.9.13
 - GPU: Nvidia RTX A6000, drivers are up to date

**Describe the bug**
This is a very weird one, as I have seen it happening on some machines and not on others. When starting Châtaigne, the user interface simply does not show up. You can only see the menu bar and the tooltips when you blindly hover something. See the following video capture.

**To Reproduce**

https://user-images.githubusercontent.com/8690430/230408821-3fb96993-e1bc-45e8-a811-80e017d551fa.mp4

**Expected behavior**
Would expect the UI to show up :-)

**Additional context**
Managed to get it working by manually copying a `Chataigne.settings` file from another (working) machine on which I had disabled _Use OpenGL Renderer_ in the _Preferences_ section. 
In other words, it looks like when the UI is using the OpenGL renderer, some machines do not display the user interface at all. For the record, same behavior can be observed with Blux on the same machine.

I also took the oportunity to enable logging in the settings before moving them to the faulty machine, but they don't show anything special regarding the UI not working.

**Comment by benkuper** [comment]
Interesting. Should openGL be disabled by default ?

**Comment by benkuper** [comment]
Would be great to debug on this computer to see if there is an error raising that we can catch to disable OpenGL automatically

**Comment by sebescudie** [comment]
Had a look at Windows' event log as well but could not see anything related to this error, will double-check next week. Also, if you think about something specific to look for, please let me know and I'll have a look :-)

When you say "debug", do you mean running Châtaigne from source to see if something shows up in VS?

**Comment by benkuper** [comment]
yes running from source :)
In case this happens again, you don't need to find a Chataigne.settings files, you can specify -forceNoGL to avoid using openGL

**Comment by AlCan13** [comment]
I have experienced this using version 1.9.24 on fresh install of Win10 x64. 
Also on Win7 GUI is very slow with OpenGL enabled: for example in Inspector when expanding/ collapsing sections like "> Parameters", "> Values" etc.
I think disabling OpenGL bu default is a good idea.
